docs/ref/contrib/gis/install/postgis.txt

```diff
@@ -7,20 +7,25 @@ into a spatial database. :ref:`geosbuild`, :ref:`proj4` and
 :ref:`gdalbuild` should be installed prior to building PostGIS. You
 might also need additional libraries, see `PostGIS requirements`_.
 
-The `psycopg2`_ module is required for use as the database adapter when using
-GeoDjango with PostGIS.
+The `psycopg`_ or `psycopg2`_ module is required for use as the database
+adapter when using GeoDjango with PostGIS.
 
 On Debian/Ubuntu, you are advised to install the following packages:
 ``postgresql-x.x``, ``postgresql-x.x-postgis``, ``postgresql-server-dev-x.x``,
-and ``python-psycopg2`` (x.x matching the PostgreSQL version you want to
+and ``python-psycopg3`` (x.x matching the PostgreSQL version you want to
```

Comment:
```suggestion
and ``python3-psycopg3`` (x matching the PostgreSQL version you want to
```
the package available on Ubuntu/Debian for `psycopg` version 3+ is actually called `python3-psycopg3`